app/controllers/api/v1/search_controller.rb

```diff
@@ -619,6 +630,27 @@ def self.match_studies_by_facet(query_results, search_facets)
         matches
       end
 
+      # Simplify TDR results to be mappable for the UI badges for facet search
```

Comment:
More conventional phrasing:

```suggestion
      # Simplify TDR results to be mappable for the UI badges for faceted search
```